test/track.cpp

```diff
@@ -33,12 +33,14 @@ void test_track() {
 	InitLogger(LogLevel::Debug);
 
 	Configuration config1;
+	config1.forceTrackAllocation = true;
```

Comment:
@paullouisageneau: with my condition above (in src/peerconnection.cpp), these tests now need `config.forceTrackAllocation`. Which suggests that either my condition above is wrong, or those tests were not really adding the track, right?

Now they fail, but I don't understand why. Would be nice to have your opinion :innocent:.

Reply:
I think this is caused by the check in `addTrack()`, see https://github.com/paullouisageneau/libdatachannel/pull/733#discussion_r1011520309